src/chapter08/exercises/exercise01.lean

```diff
@@ -6,10 +6,39 @@ change, any whole number of roubles greater than 7 with
 banknotes of value 3 roubles and 5 roubles
 -/
 
+
 example (n : ℕ) (hn : 7 < n) : ∃ a b : ℕ, n = 3 * a + 5 * b :=
 begin
   -- Remark: this looks a little subtle to me, you will have
   -- to think about exactly what statement you want to prove
   -- by induction.
-  sorry
-end
\ No newline at end of file
+  generalize hd : n - 8 = d,
+  have hnd : n = d + 8,
+  { linarith, },
+  rw hnd,
+  clear hnd hd hn n,
+  induction d with k hk,
+  { use 1, use 1, norm_num, },
+  change ∃ a b, k + 1 + 8 = 3 * a + 5 * b,
```

Comment:
...this line should be 
```suggestion
  { change ∃ a b, k + 1 + 8 = 3 * a + 5 * b,
```
and all the lines below it should be indented two more spaces, and the last line should end with }.